azkaban-webserver/src/main/resources/azkaban/webapp/servlet/velocity/executingflowpage.vm

```diff
@@ -112,6 +112,18 @@
         <li class="nav-button pull-right"><button type="button" id="cancelbtn" class="btn btn-danger btn-sm">Kill</button></li>
         <li class="nav-button pull-right"><button type="button" id="retrybtn" class="btn btn-success btn-sm">Retry Failed</button></li>
         <li class="nav-button pull-right"><button type="button" id="executebtn" class="btn btn-success btn-sm">Prepare Execution</button></li>
+        #*
+         * The 'pull-right' bootstrap class will cause the first 'li' element
+         * to take the right most spot. As 'analyzerButton' is mentioned in the
+         * last line, it will appear as the first button on the right.
+         *
+         * Tab focus is disabled on 'analyzerButton' by setting tabindex to -1 
+         * for the following reason. Whenever two buttons are displayed on the
```

Comment:
It will look like the following. Not too bad IMO.
I will go ahead with this.

![image](https://cloud.githubusercontent.com/assets/11494855/15682920/75ab0a2a-277d-11e6-8f85-3a00fcb2f30e.png)